studio_app/templates/layout.html

```diff
@@ -16,59 +16,7 @@
 <link href="{{ url_for('static', filename='/css/bootstrap.min.css') }}" rel="stylesheet">
 <link href="{{ url_for('static', filename='/css/style.css') }}" rel="stylesheet">
 
-    <style>
-
-      .bd-placeholder-img {
-        font-size: 1.125rem;
-        text-anchor: middle;
-        -webkit-user-select: none;
-        -moz-user-select: none;
-        user-select: none;
-      }
-
-      @media (min-width: 768px) {
-        .bd-placeholder-img-lg {
-          font-size: 3.5rem;
-        }
-      }
-
-      .b-example-divider {
-        height: 3rem;
-        background-color: rgba(0, 0, 0, .1);
-        border: solid rgba(0, 0, 0, .15);
-        border-width: 1px 0;
-        box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
-      }
-
-      .b-example-vr {
-        flex-shrink: 0;
-        width: 1.5rem;
-        height: 100vh;
-      }
-
-      .bi {
-        vertical-align: -.125em;
-        fill: currentColor;
-      }
-
-      .nav-scroller {
-        position: relative;
-        z-index: 2;
-        height: 2.75rem;
-        overflow-y: hidden;
-      }
-
-      .nav-scroller .nav {
-        display: flex;
-        flex-wrap: nowrap;
-        padding-bottom: 1rem;
-        margin-top: -1px;
-        overflow-x: auto;
-        text-align: center;
-        white-space: nowrap;
-        -webkit-overflow-scrolling: touch;
-      }
-    </style>
+    <link href="{{ url_for('static', filename='/css/layout.css') }}" rel="stylesheet">
     {% block custom_styles %}
```

Comment:
I noticed that in other areas of the code-base the link to custom CSS is under {% block custom_styles %}. I just put the link in the same place that I saw it but I do think that this line should be under {% block custom_styles %}

The line I'm referring to is : <link href="{{ url_for('static', filename='/css/layout.css') }}" rel="stylesheet">